internal/controller/validator.go

```diff
@@ -119,6 +119,17 @@ func (r *DataProtectionApplicationReconciler) ValidateDataProtectionCR(log logr.
 				return false, fmt.Errorf("DPA spec.nonAdmin.garbageCollectionPeriod can not be negative")
 			}
 		}
+
+		enforcedBackupSpec := r.dpa.Spec.NonAdmin.EnforceBackupSpec
+
+		if enforcedBackupSpec != nil {
```

Comment:
here we need to decide where to validate this

currently, this is done on NAC side https://github.com/migtools/oadp-non-admin/blob/master/internal/common/function/function.go#L98-L102

Reply:
Hmm EnforcedSpec is done by Admins and specified on DPA so I think we should move the validation to OADP Operator

Reply:
I agree :)

Reply:
lets move this https://github.com/migtools/oadp-non-admin/blob/7b6f8b35a0e40955a3151ea7902e162489e72161/internal/common/function/function.go#L98-L102 to here in this PR?

Reply:
Ah yeah I was thinking about that yesterday, I can update the PR

Reply:
Done, updated the PR

Reply:
do not forget to remove that code on NAC side

Reply:
added it in this PR https://github.com/migtools/oadp-non-admin/pull/208